migration/v0.31/README.md

```diff
@@ -132,6 +144,12 @@ As with all scripts in this repository `CK8S_CONFIG_PATH` is expected to be set.
     ./migration/v0.31/apply/20-bootstrap.sh execute
     ```
 
+1. Remove Openseach configurer:
+
+    ```bash
+    ./migration/v0.31/apply/11-remove-opensearch-configurer.sh execute
+    ```
+
```

Comment:
This step already exists, and a lot of the steps here are not written in the same order as the number of the snippets.